src/operations/groupBy.ts

```diff
@@ -48,12 +51,27 @@ export function groupValuesByColumn({
   const targetOperation =
     aggregationFunctions[operation as Exclude<AggregationType, 'custom'>];
 
-  if (targetOperation) {
-    return Array.from(groups).map(([name, value]) => ({
-      name,
-      value: targetOperation(value),
-    }));
+  if (!targetOperation) {
+    return [];
   }
 
-  return [];
+  const allCategories = Array.from(groups).map(([name, value]) => ({
+    name,
+    value: targetOperation(value),
+  }));
+
+  allCategories.sort((a, b) => b.value - a.value);
```

Comment:
If inmutability is preferred (I guess so?) we can chain the previous `.map` function with the `.sort` one